docs/LayoutSpecs.json

```diff
@@ -375,7 +375,8 @@
               "path": "DSCore.DateTime.TimeOfDay"
             },
             {
-              "path": "DSCore.DayOfWeek"
+              "path": "DSCore.DayOfWeek",
+              "pathIcon": "/src/resources/icons/DSCore.DayOfWeek.png"
```

Comment:
Hi @yeexinc, do you mean if we don't specify `pathIcon` here, then no icon will be rendered for `DSCore.DayOfWeek` on the library view? I'm trying to see if `pathIcon` is absolutely necessary. I think we need it here because the location of the icons need to be absolute path, correct me if I'm wrong. In that case, can we rename `pathIcon` to `iconUrl` just to be consistent?